Ruby is not installed in Docker base image

https://console.cloud.google.com/cloud-build/builds/327a61a6-f2cb-4bc3-b81d-2b821cd8731c?project=dcap-sapphire-github-ci

```
> Compilation failed; see the compiler error output for details.
Execution failed for task ':examples:hanksTodoRuby:compileJava'.
* What went wrong:

FAILURE: Build failed with an exception.

FAILED
2 errors
^
tls = sapphire.appexamples.hankstodo.stubs.TodoListManager_Stub.getStub(fileName,
/DCAP-Sapphire/sapphire/examples/hanksTodoRuby/src/main/java/sapphire/appexamples/hankstodo/Client.java:26: error: package sapphire.appexamples.hankstodo.stubs does not exist
^
sapphire.appexamples.hankstodo.stubs.TodoListManager_Stub tls;
:examples:hanksTodoRuby:compileJava/DCAP-Sapphire/sapphire/examples/hanksTodoRuby/src/main/java/sapphire/appexamples/hankstodo/Client.java:24: error: package sapphire.appexamples.hankstodo.stubs does not exist

INFO: Could not find class TodoListManager in language ruby
Nov 30, 2018 12:17:08 AM sapphire.compiler.GraalStubGenerator generateStub
INFO: Could not find class TodoListManager in language python
Nov 30, 2018 12:17:08 AM sapphire.compiler.GraalStubGenerator generateStub
INFO: Could not find class TodoListManager in language js
Nov 30, 2018 12:17:08 AM sapphire.compiler.GraalStubGenerator generateStub
WARNING: Failed to evaluate file /DCAP-Sapphire/sapphire/examples/hanksTodoRuby/src/main/ruby/sapphire/appexamples/hankstodo/todo_list_manager.rb via lang ruby, exception is A language with id 'ruby' is not installed. Installed languages are: [js, llvm].
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,14 +1,9 @@
 
-FROM dcap/sapphire_base:0.1
+FROM dcap/sapphire_base:0.2
 
 COPY ./ /DCAP-Sapphire/
 
 WORKDIR /DCAP-Sapphire/sapphire
-
-# Install GraalVM ruby component
-# NOTE: This is a temporary measure until dcap/sapphire_base:0.2 has been updated.
-# Once that has been done, the following command should be removed.
-RUN bash gu install ruby
 
 # Verifying the build
 RUN bash gradlew build

